### DISCUSSION TITLE
Flyby11 0.15

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/FlyOOBE#13] [Announcements]
- Added the **USB Compatibility Patch** option to improve Windows 11 bootable USB drives. Now, you can apply a system requirement bypass to any bootable USB drive, including those made with third-party tools like Rufus. For example, you could create your bootable USB stick using the Media Creation Tool, and then apply this compatibility patch directly, regardless of the built-in method from Flyby11 or other tools. You can find this option under "I want to create Windows 11 Installation media."
- Also, I added in 0.14 a new script to make upgrading to Windows 11 24H2 easier for officially supported systems, simplifying the update process. Be sure to use (Direct) Method 3, as this cuts down the wait time for the update and allows for an immediate upgrade. You can find all the details in the Patch3.json file located in the "upgraider" folder.

![explorer_BH0uv8i4Ro](https://github.com/user-attachments/assets/310f8287-a056-45b1-b5c8-8e223ab44ad4)

### Some of you want to thank me, so [I’ll drop my PayPal link here](https://paypal.com/donate?hosted_button_id=MY7HX4QLYR4KG). Thanks a lot, friends, and see you soon!

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Flyby11/releases/tag/0.15'>Flyby11 0.15</a>.</em>

**Comment by Beber17** [comment]
Bonjour ou Bonsoir !
Si j'installe votre patches quelle est le risque des logiciel tiers déjà installer sous Windows 10 ?

Beber17

**Comment by itdanielb** [comment]
Hi, thanks for this great project!

I created a Win11 24H2 bootable USB using different tools (Microsoft, Rufus. and Flyby11) and Added the USB Compatibility Patch. It boots, but after a few clicks, it shows that the system is not compatible with Windows 11. Is there any step I'm missing?

Please Advise.
